src/main/paradox/features/applications.md

```diff
@@ -0,0 +1,45 @@
+## Applications and Jobs
+
+You can declare additional *applications* inside your Docker image. An application is a name and a sequence of arguments
+that should be executed for the application. An operator can then generate alternative resources using `rp` by simply
+declaring the application name.
+
+On Kubernetes, this is complemented by the support for generating `Job` resources. For example, the following configuration would
+result in the main method defined by `App` being run by default. However, the operator can invoke `rp generate-kubernetes-resources --application my-job`
+to generate resources for the defined application instead.
+
+Given the following build configuration:
+
+```sbt
+mainClass in Compile := Some("com.example.App")
+
+applications += "my-job" -> Vector("bin/myjob")
+
+```
+
+And, given the following main classes:
```

Comment:
`objects`

do we not document java?

Reply:
https://github.com/lightbend/lightbend-orchestration-docs/issues/56